github/dependabot_secrets.go

```diff
@@ -144,8 +145,29 @@ func (s *DependabotService) CreateOrUpdateRepoSecret(ctx context.Context, owner,
 //
 // GitHub API docs: https://docs.github.com/en/rest/dependabot/secrets#create-or-update-an-organization-secret
 func (s *DependabotService) CreateOrUpdateOrgSecret(ctx context.Context, org string, eSecret *DependabotEncryptedSecret) (*Response, error) {
+	repoIDs := make([]string, len(eSecret.SelectedRepositoryIDs))
+	for i, secret := range eSecret.SelectedRepositoryIDs {
+		repoIDs[i] = strconv.Itoa(int(secret))
```

Comment:
```suggestion
		repoIDs[i] = fmt.Sprintf("%v", secret)
```

Reply:
No, let's please change it back to `%v` because it is idiomatic Go to use `%v` unless there is a specific reason otherwise, such as if we need leading zeros, as in `%05d`.
Although you may be able to find other examples in this repo where this is a `%d`, we have attempted to remain consistent and use `%v` unless there is a specific need.  Thanks.